go.mod

```diff
@@ -2,6 +2,12 @@ module github.com/fluxcd/flux2
 
 go 1.18
 
+// Use Flux's version of go-git, which allows clones with multi_ack and multi_ack_detailed.
```

Comment:
Now that `fluxcd/pkg` migrated to `fluxcd/go-git`, we should update that here to. That can be a commit in itself.